### PR TITLE
clarifying owners can delete

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8593

image

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@tiburd Can you help me to get this merged?

File: Teams/messaging-policies-in-teams.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ A user can only be assigned one messaging policy at a time.
 
 Here are the messaging policy settings that you can configure.
 
-- **Owners can delete sent messages**  Use this setting to let owners delete messages that users sent in chat.
+- **Owners can delete sent messages**  Use this setting to let owners delete channel messages/posts that users sent.
```
Comment (tiburd) [inline Teams/messaging-policies-in-teams.md:73]:
```suggestion
- **Owners can delete sent messages**  Use this setting to let owners delete channel messages or posts that users sent.
```

**Pull request merged**